src/event.c

```diff
@@ -213,7 +221,7 @@ void process_input(struct device *dev, struct dev_input *inp)
 	}
 }
 
-static void handle_button_action(int act)
+static void handle_button_action(int act, int val)
```

Comment:
instead of "val", I'd just call this "pressed". And then just check for `if(pressed)` inside. Also the sensitivity adjustment was not supposed to happen on both press and release. That was a bug. But I'll fix that later.

Reply:
ok, got it